tests/test_actions.py

```diff
@@ -435,6 +436,19 @@ def new_method(self) -> bool:
 
         assert SubEnter.the_text("blah").new_method() is True
 
+    def test_positional_arg_warns(self) -> None:
+        with pytest.warns(DeprecationWarning):
+            Enter("", True)
+
+    def test_keyword_arg_does_not_warn(self) -> None:
+        with warnings.catch_warnings():
+            warnings.simplefilter("error")
+            Enter.the_secret("")
+
+        with warnings.catch_warnings():
+            warnings.simplefilter("error")
+            Enter("", mask=True)
```

Comment:
I don't think you're actually testing anything here, i think you need to add a `as caught_warnings` to the context manager and then do some assertions on it later. See the docs here: https://docs.python.org/3/library/warnings.html#testing-warnings

Reply:
This same comment applies to the other tests below.

Reply:
This test works, albeit it's proving a negative.  It's testing to ensure the warning does NOT occur when `mask` keyword is used. 

You can try it out by removing the keyword.
```python
# THIS passes (as expected)
        with warnings.catch_warnings():
            warnings.simplefilter("error")
            Enter("", mask=True)
```
```python
# but THIS will fail 
        with warnings.catch_warnings():
            warnings.simplefilter("error")
            Enter("", True)
```

Reply:
OK, i see now that we get an error for the raised warning, but i don't like that the error _is_ the warning, you know? I'd prefer an assertion that makes it clear what the problem was. Is that possible to do with what they've got in the docs?

I guess it'll only be relevant for a couple months, so i think i can be OK with this.

Reply:
I updated the tests.  They explicitly state when a warning was raised unexpectedly, rather than only raising the warning.